server.go

```diff
@@ -34,6 +40,8 @@ type Server struct {
 	tlsErrors                   metrics.Counter
 }
 
+const keepAlive = 3 * time.Minute
```

Comment:
use old default for keep alive see previous version https://github.com/golang/go/blob/go1.12/src/net/http/server.go#L3269